db/seeds/BaseStructureAndData.php

```diff
@@ -932,6 +932,11 @@ public function run(): void {
             (5, 'Mist', '16.10', 0, 0);"
         );
 
+        // System Storage
+    $this->execute(
+        "INSERT INTO `system_storage` (`global_message`, `time`) VALUES ('', '" . time() . "');"
+    );
```

Comment:
Intended to for future seeds. I think most people running local environments won't really notice this, since they're not using global messages/maintenance.

Reply:
What would happen if we didn't have this line?

Reply:
Game breaks if a global is sent out and server maintenance fails.
I suppose it could be added via a migration as a truncate => add row.

Reply:
Ah, I see. Hmm... Okay, I think we can say this is a special case since we manually applied this already and everyone that's already set up a dev environment would have had to manually set it up too. Let's leave it in the seed file 👍